packages/next/server/app-render.tsx

```diff
@@ -1598,11 +1598,35 @@ export async function renderToHTMLOrFlight(
         ).slice(1),
       ]
 
+      const serverComponentManifestWithHMR = dev
+        ? new Proxy(serverComponentManifest, {
+            get: (target, prop) => {
+              if (
+                typeof prop === 'string' &&
+                !prop.startsWith('_') &&
+                target[prop]
+              ) {
+                // Attach TS query param to IDs to get rid of flight client's module cache on HMR.
```

Comment:
```suggestion
                // Attach TS (timestamp) query param to IDs to get rid of flight client's module cache on HMR.
```

Reply:
When adding this property is there a chance the client-side module cache will keep growing indefinitely?